doc/compiler/jitserver/Usage.md

```diff
@@ -49,22 +51,25 @@ You might have noticed that running the client without any server to connect to
 
 With verbose logging, if a client connects successfully then server output should look something like this:
 ```
-$ java -XX:StartAsJITServer -Xjit:verbose=\{JITServer\}
-
-#JITServer: JITServer in Server Mode. Port: 38400
-#JITServer: Started JITServer listener thread: 0000000001B8DE00
-JITServer ready to accept incoming requests
-
-#JITServer: Server allocated data for a new clientUID 7273413081043723869
-#JITServer: Server received request to compile sun/reflect/Reflection.getCallerClass @ cold
-#JITServer: Server queued compilation for sun/reflect/Reflection.getCallerClass
-#JITServer: Server cached clientSessionData=00007FFFDA1973E0 for clientUID=7273413081043723869 compThreadID=0
-#JITServer: Server has successfully compiled sun/reflect/Reflection.getCallerClass()Ljava/lang/Class;
-#JITServer: Server received request to compile java/lang/Double.longBitsToDouble @ cold
-#JITServer: Server queued compilation for java/lang/Double.longBitsToDouble
-#JITServer: Server cached clientSessionData=00007FFFDA1973E0 for clientUID=7273413081043723869 compThreadID=1
-#JITServer: Server has successfully compiled java/lang/Double.longBitsToDouble(J)D
-#JITServer: Server received request to compile java/lang/System.getEncoding @ cold
+$ jitserver -Xjit:verbose=\{JITServer\}
+
+JITServer is currently a technology preview. Its use is not yet supported
+
+#JITServer: JITServer Server Mode. Port: 38400. Connection Timeout 2000ms
+#JITServer: Started JITServer listener thread: 0000000000226C00
+JITServer is ready to accept incoming requests
+
+#JITServer: Server received request for stream 00007FEC658EF720
+#JITServer: Server allocated data for a new clientUID 11129135271614904954
+#JITServer: compThreadID=0 created clientSessionData=00007FEC658EFBE0 for clientUID=11129135271614904954 seqNo=0
+#JITServer: Server will process a list of 0 unloaded classes for clientUID 11129135271614904954
+#JITServer: compThreadID=0 has successfully compiled java/lang/Double.longBitsToDouble(J)D
+#JITServer: compThreadID=0 found clientSessionData=00007FEC658EFBE0 for clientUID=11129135271614904954 seqNo=1
+#JITServer: Server will process a list of 0 unloaded classes for clientUID 11129135271614904954
+#JITServer: compThreadID=0 has successfully compiled jdk/internal/reflect/Reflection.getCallerClass()Ljava/lang/Class;
+#JITServer: compThreadID=0 found clientSessionData=00007FEC658EFBE0 for clientUID=11129135271614904954 seqNo=2
+#JITServer: Server will process a list of 0 unloaded classes for clientUID 11129135271614904954
+#JITServer: compThreadID=0 has successfully compiled java/lang/System.getEncoding(I)Ljava/lang/String;
```

Comment:
Below, when specifying the server address the correct syntax is
`java -XX:UseJITServerAddress=example.com`